File: app/src/main/java/com/druger/refresher/model/ModelTask.java
package com.druger.refresher.model;

/**
 * Created by druger on 16.09.2015.
 */
public class ModelTask implements Item {

    private String title;
    private long date;

    public ModelTask() {
    }

    public ModelTask(String title, long date) {
        this.title = title;
        this.date = date;
    }

    @Override
    public boolean isTask() {
        return true;
    }

    public long getDate() {
        return date;
    }

    public void setDate(long date) {
        this.date = date;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

}

File: app/src/main/java/com/druger/refresher/domain/task/model/ModelTask.kt
package com.druger.refresher.domain.task.model

import com.druger.refresher.data.db.entity.Task
import java.util.*

data class ModelTask(
    val title: String,
    val timeStamp: Long = Date().time,
    var status: Int = Task.STATUS_CURRENT,
    val reminderDate: Long = 0,
    val id: Int = 0
)

fun ModelTask.map() = Task(id, title, timeStamp, status, reminderDate)